ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -876,38 +880,35 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           populationDetails.includes(SpecificPopulationEnum.OTHER)) {
         values = {...values, otherPopulationDetails};
         constraints['otherPopulationDetails'] = {
-          presence: {
-            allowEmpty: false
-          },
           length: {
-            maximum: 100
+            minimum: 1,
+            maximum: 100,
+            ...lengthMessages('Specific Population Other')
           }
         };
       }
       if (diseaseFocusedResearch) {
         values = {...values, diseaseOfFocus};
         constraints['diseaseOfFocus'] = {
-          presence: {
-            allowEmpty: false
-          },
           length: {
-            maximum: 80
+            minimum: 1,
+            maximum: 80,
+            ...lengthMessages('Disease Of Focus')
           }
         };
       }
       if (disseminateResearchFindingList &&
           disseminateResearchFindingList.includes(DisseminateResearchEnum.OTHER)) {
         values = {...values, otherDisseminateResearchFindings};
         constraints['otherDisseminateResearchFindings'] = {
-          presence: {
-            allowEmpty: false
-          },
           length: {
-            maximum: 100
+            minimum: 1,
+            maximum: 100,
+            ...lengthMessages('Disseminate Research Findings Other')
```

Comment:
nit: this should probably be something like `Other methods of disseminating research findings`